test/modules/storage-aws-s3/snapshot_create_test.go

```diff
@@ -42,10 +42,40 @@ func Test_S3Storage_StoreSnapshot(t *testing.T) {
 		s3Config := s3.NewConfig(endpoint, "bucket", false)
```

Comment:
you can now put this piece of code out of this `t.Run(...)` and put it directly inside `Test_S3Storage_StoreSnapshot` so that to indicate that the tests for `store` and `restore` use the same credentials to authenticate against `MinIO`

Reply:
I'm refering to this:
```
os.Setenv("AWS_REGION", "eu-west-1")
os.Setenv("AWS_ACCESS_KEY", "aws_access_key")
os.Setenv("AWS_SECRET_KEY", "aws_secret_key")
endpoint := os.Getenv(minioEndpoint)
```